on-boarding/task-2/main.dart

```diff
@@ -89,8 +107,14 @@ void main() {
         break;
 
       case 6:
-        print('Enter task title of task you want to delete:');
-        String title = stdin.readLineSync() ?? '';
+        String title = '';
```

Comment:
Missing Break Statement:
In the 'Delete task' case, there is a missing break statement after deleting the task. Adding the break statement will prevent the application from continuing to execute the 'Edit task' case after successfully deleting a task.